dask/dataframe/io/parquet/arrow.py

```diff
@@ -897,11 +897,6 @@ def _collect_dataset_info(
         # be avoided at all costs.
```

Comment:
^ There's a doc string above commenting on the split_row_groups behavior, but that seems to have been removed here? Should that doc string still be here?

Reply:
Do you mean the comment starting on line 892, or a docstring on a method somewhere?